test/utils/describeConformanceV5.js

```diff
@@ -21,11 +21,18 @@ import {
 function testComponentsProp(element, getOptions) {
   describe('prop: components', () => {
     it('can render another root component with the `components` prop', () => {
-      const { classes, mount, testComponentsRootPropWith: component = 'em' } = getOptions();
-
-      const wrapper = mount(React.cloneElement(element, { components: { Root: component } }));
-
-      expect(findRootComponent(wrapper, { classes, component }).exists()).to.equal(true);
+      const {
+        classes,
+        mount,
+        testComponentsRootPropWith: component = 'em',
+        skipComponentsProp,
+      } = getOptions();
+
+      if (!skipComponentsProp) {
```

Comment:
I believe the `skip` API is already meant to solve this problem. Do we need a new option?

Reply:
I wasn't aware of it :) Reverted those changes 👍